shielddb-core/src/test/java/nf/fr/k49/shielddb/core/storage/FileStorageInstantiationTest.java

```diff
@@ -0,0 +1,88 @@
+package nf.fr.k49.shielddb.core.storage;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Comparator;
+import java.util.UUID;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileStorageInstantiationTest {
+
+    static Path workDirectoryForTests;
+
+    @BeforeClass
+    public static void beforeAll() throws Exception {
+        workDirectoryForTests = Files.createTempDirectory(FileStorage.class.getSimpleName());
+    }
+
+    @AfterClass
+    public static void afterAll() throws Exception {
+        Files.walk(workDirectoryForTests).sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
```

Comment:
why did you sorting files before deleting them?

Reply:
Sorry... I did a mistake... I've worked later of the night then I was tired a little bit... I'll fix it!